Doc set for VMFR (#110937)
openshift/openshift-docs: backup_and_restore/application_backup_and_restore/oadp-vmfr/oadp-using-vmfr.adoc

:_mod-docs-content-type: ASSEMBLY
include::_attributes/common-attributes.adoc[]

[id="oadp-configuring-using-vmfr"]
= Configure and use {oadp-short} VMFR
:context: oadp-using-vmfr

toc::[]

[role="_abstract"]

> Use {oadp-short} virtual machine file restore (VMFR) to discover virtual machine backups, restore files, and access restored files through a web browser or SSH-based tools. This helps you to recover individual files from VM backups without restoring the entire virtual machine.

Reviewer: ```suggestion
Use {oadp-short} virtual machine (VM) file restore (VMFR) to discover VM backups, restore files, and access restored files through a web browser or SSH-based tools. You can recover individual files from VM backups without restoring the entire VM.

```
